tests/runner/Testcontracts.ml

```diff
@@ -153,10 +162,11 @@ let add_tests env =
   "contract_tests" >:::[
     "these_tests_must_SUCCEED" >:::[
       "crowdfunding" >:::(build_contract_tests env "crowdfunding" succ_code 1 6 false);
-      "crowdfunding_init" >:(build_contract_init_test env "crowdfunding" false);
+      "crowdfunding_init" >:(build_contract_init_test env succ_code "crowdfunding" false);
       "zil-game" >:::(build_contract_tests env "zil-game" succ_code 1 9 false);
-      "zil-game_init" >:(build_contract_init_test env "zil-game" false);
-      "testlib2_init" >:(build_contract_init_test env "TestLib2" true);
+      "zil-game_init" >:(build_contract_init_test env succ_code "zil-game" false);
+      "testlib1_init" >:(build_contract_init_test env fail_code"0x565556789012345678901234567890123456abcd" true);
```

Comment:
Please add a space before the first double quote... :-)

Also, why is `testlib1_init` listed in the "these_tests_must_SUCCEED" section?

Reply:
thanks for catching both of these, I'll fix them.

Reply:
Fixed